### ISSUE TITLE
please can you create example ?

### ISSUE THREAD

**Issue by Khalidalansi** [hoxfon/react-native-twilio-programmable-voice#100]
Please include the following information for better support

The more context you provide around this issue the faster the community can help you

Did you follow all the instructions as specified in the Twilio Quickstart repositories?
What version of React Native are you running?
What version of react-native-twilio-programmable-voice are you running?
What device are you using? (e.g iOS10 simulator, Android 7 device)?
Is your app running in foreground, background or not running?
Is there any relevant message in the log?
If using iOS, which pod version are you using?

Step to reproduce

Advanced:
Have you tried adding break point using AndroidStudio or XCode and analyse the logs?
can share a project with issue?
Did you try to reinstall the pods completely?

**Comment by alfonsosn** [comment]
Agreed, I've tried using this library in the past with iOS and the setup never works.

**Comment by edzh1** [comment]
@Khalidalansi @alfonsosn Hi. I can provide a gist with example. What do you need exactly?

**Comment by Khalidalansi** [comment]
@edzh1  Thank you for your reply.  I would like to integrate these capabilities into a React Native app:

1.  Receive calls through the app itself; not have to ring person’s actual number
2.  Make calls through the app itself
3.  Make and receive texts in app
4.  Record all calls and show list of recent calls in app

**Comment by Khalidalansi** [comment]
@edzh1 Yes, that will be great!  I am looking forward to it.

**Comment by edzh1** [comment]
@Khalidalansi Please, give me a couple days to create an example app and medium article with an explanation. I will notify you when it'll be done

**Comment by Khalidalansi** [comment]
@edzh1 Thank you!!

**Comment by fabriziomoscon** [comment]
In principle I second the idea, but in my experience the example apps I saw in other libraries in the past are constantly out of date. And the reason is that authors are always in short of time and react-native evolves very quickly. 

I currently don't have capacity for an example app. Honestly I will not have capacity to keep it updated with the latest ever improving react native and Google libraries.
If you have capacity for an example app please include a complete explanation on which compatibility libraries are used, so that the user are aware when is out of date.

Thanks!

**Comment by edzh1** [comment]
@Khalidalansi @alfonsosn https://medium.com/@edzh1/create-a-twilio-voip-calls-in-a-react-native-app-35a729a9613d

**Comment by alfonsosn** [comment]
@edzh1 I'll be working on integrating the library in the following days, but thank you so much for this super helpful guide, really clarified things for me. LEGEND!

**Comment by edzh1** [comment]
> @edzh1 I'll be working on integrating the library in the following days, but thank you so much for this super helpful guide, really clarified things for me. LEGEND!

Glad I could help :)

**Comment by marcelocbf** [comment]
> @Khalidalansi @alfonsosn https://medium.com/@edzh1/create-a-twilio-voip-calls-in-a-react-native-app-35a729a9613d

Wouldn't be able to integrate the library without it. Should be on the README. Thank you so much!

**Comment by gentleShark** [comment]
@edzh1 @fabriziomoscon does anyone have an example of how to accept a call on Android? presumably on iOS this is handled by CallKit? I can get my emulator to ring but I'm looking for a way to capture the event and present the user with the option to accept or reject the call

**Comment by jdegger** [comment]
In the next couple of months we will be trying to improve the docs and add examples. If somebody is willing to help, read our update here on how to contribute #158

**Comment by morpheis91** [comment]
Please if you could expedite this or have an ongoing one as PRs get merged.  Do you know if this example is still valid? 
[](https://medium.com/@edzh1/create-a-twilio-voip-calls-in-a-react-native-app-35a729a9613d
)

**Comment by jdegger** [comment]
@morpheis91 that articile seems to be outdated. Since it was written, we have changed a lot in the library. Especially version 4 which was just released. So far nobody is actively working on the example app but we will need it in the near future.

**Comment by edzh1** [comment]
@jdegger I can update my article and create new example app

**Comment by jdegger** [comment]
@edzh1 that would be wonderful. Would you be interested in authoring the example app and publishing it on the hoxfon org so we can maintain it here and use for testing as well? In #158 I also explain that we are looking for somebody to help us with setting up a proper example

**Comment by edzh1** [comment]
@jdegger sure, will start working on that

**Comment by danstepanov** [comment]
@edzh1 Did you ever end up finishing the example?
